Libraries/LibWeb/HTML/Scripting/ImportMap.cpp

```diff
@@ -213,10 +213,8 @@ WebIDL::ExceptionOr<HashMap<URL::URL, ModuleSpecifierMap>> sort_and_normalise_sc
         }
 
         // 4. Let normalizedScopePrefix be the serialization of scopePrefixURL.
-        auto normalised_scope_prefix = scope_prefix_url->serialize();
-
         // 5. Set normalized[normalizedScopePrefix] to the result of sorting and normalizing a module specifier map given potentialSpecifierMap and baseURL.
-        normalised.set(normalised_scope_prefix, TRY(sort_and_normalise_module_specifier_map(realm, potential_specifier_map.as_object(), base_url)));
+        normalised.set(scope_prefix_url.value(), TRY(sort_and_normalise_module_specifier_map(realm, potential_specifier_map.as_object(), base_url)));
```

Comment:
Just wondering out lout - is there a case where we are now producing a different result than we would have by going from URL -> string -> URL? I'm assuming not, but seems strange that the spec has us doing that (unless the spec things `normalized` is keyed by strings, and we key by URL instead?)

Reply:
Indeed, the spec keys it by strings, but we look to have written it by using URL record. The implicit conversion beforehand made this look less weird than it was 😅 

I don't believe there is any functional difference in using URL record as the key since `URL::URL` equality is same as equality of the serialization